backend/s3/s3.go

```diff
@@ -1180,12 +1180,18 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// Initialize pacer to 0 retries because we are relying on SDK with
+	// MaxRetries for that
+	pc := fs.NewPacer(pacer.NewS3(pacer.MinSleep(minSleep)))
+	pc.SetRetries(1)
```

Comment:
1 vs 0

Reply:
It doesn't work with 0. Attempts would be a better naming in this case.